types/ioredis/index.d.ts

```diff
@@ -50,18 +50,33 @@ declare class Commander {
     sendCommand(): void;
 }
 
+interface CommandOptions {
+    replyEncoding?: string | null;
+    errorStack?: string;
+    keyPrefix?: string;
+}
+declare class Command {
+    isCustomCommand: boolean;
+    args: IORedis.ValueType[];
+    getSlot(): number | null;
+    getKeys(): Array<string | Buffer>;
+    constructor(
+        name: string,
+        args: IORedis.ValueType[],
+        opts?: CommandOptions,
+        callback?: (err: null, result: any) => void,
+    );
+    static setArgumentTransformer(name: string, fn: (args: IORedis.ValueType[]) => IORedis.ValueType[]): void;
+    static setReplyTransformer(name: string, fn: (result: any) => any): void;
+}
+
 declare namespace IORedis {
     type KeyType = string | Buffer;
 
     type BooleanResponse = 1 | 0;
 
     type ValueType = string | Buffer | number | any[];
 
-    interface Command {
```

Comment:
You should _probably_ add a `type _Command = typeof Command` outside the IORedis namespace, and then a `type Command = _Command` inside the namespace, so as to not break external users who referenced the `Command` interface, no?

Reply:
Thanks @weswigham. Good catch, and I've pushed that change. I was thinking about how/whether to avoid a break actually, but couldn't immediately figure out how to get it to work with namespaces.